packages/modeling/src/operations/hulls/quickhull/point-line-distance.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * point-line-distance
+ *
+ * Copyright (c) 2015 Mauricio Poppe
+ * Licensed under the MIT license.
+ */
+
+const cross = require('../../../math/vec3/cross')
+const subtract = require('../../../math/vec3/subtract')
+const squaredLength = require('../../../math/vec3/squaredLength')
+
+const distanceSquared = (p, a, b) => {
```

Comment:
should we perhaps move this one to maths at some point ? I can imagine it being useful